nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/CommentManagerImpl.java

```diff
@@ -500,4 +458,48 @@ public boolean hasFeature(Feature feature) {
             throw new UnsupportedOperationException(feature.name());
         }
     }
+
+    @Override
+    public List<DocumentModel> getComments(CoreSession session, DocumentModel docModel) {
```

Comment:
For the same reason about historical, can you keep the method at the place it was?

Reply:
I had to put back the throws CommentSecurityException for the method to stay the same.
So I commented inline : 
// throw declaration is kept for historical reasons

Reply:
In 10.3 `CommentManager` introduces a new method:
`public List<DocumentModel> getComments(CoreSession session, DocumentModel docModel)`
In the `PropertyCommentManager` a security check is performed to determine if the session has read permission on the ancestor document.
In `CommentManagerImpl` there is no such security check.
We discussed it during the grooming and @troger  concluded to finally leave it like that as it’s a pure java code issue. It’s planned to discuss it with you @kevinleturc  on Monday